hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -126,6 +125,7 @@ def _default_employee(self):
         comodel_name='res.company',
         string='Company',
         default=lambda self: self.env['res.company']._company_default_get(),
+        required=True,
```

Comment:
Are you sure that doing this now in production won't create problems? 🤔

Reply:
This field was always readonly since your very first commit, I just checked the commit history. The default value `_company_default_get()` ensures that the value is always set.

It shouldn't be any problem in production, unless someone changed the value directly in the DB or in case some very particular custom module allowed the user to remove the company from the timesheet. So IMO is not creating problems.